docs/business.yaml

```diff
@@ -239,6 +322,11 @@ paths:
                   value:
                     errorMessage: API backend third party service error.
                     rootCause: "errors:<comma delimited list of potential validation errors by json path>"
+                change-of-address-missing-records-office-response:
+                  summary: Change of address - Missing records office address response
```

Comment:
Forgot to mention, also the naming here, and the `rootCause` can use the shorter version.

Reply:
this updated now